src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/items/MythicCustomItem.kt

```diff
@@ -120,7 +121,12 @@ internal data class MythicCustomItem(
                 repairCost = configurationSection.getInt("repair-cost", DEFAULT_REPAIR_COST),
                 isEnchantmentsRemovableByGrindstone = isEnchantmentsRemovableByGrindstone,
                 isAddDefaultAttributes = isAddDefaultAttributes,
-                hdbId = configurationSection.getNonNullString("hdb-id")
+                hdbId = configurationSection.getNonNullString("hdb-id"),
+                rgb = CustomItem.Rgb(
+                    red = configurationSection.getInt("rgb.red"),
```

Comment:
We should probably use a default of `-1` on these, as `0` is technically a valid value for RGB.